client/src/components/Map/InfoWindow/styles.css

```diff
@@ -36,7 +36,7 @@
   position: relative;
   width: 100%;
   background-color: white;
-  box-shadow: 0px 0px 7px 2px rgba(0, 0, 0, 0.3);
+  border-radius: 10px;
```

Comment:
we can also add `color: #333` so the text "You need to ..." in the profile page becomes more visible

Reply:
thanks for the suggestions, both should be all fixed up now